src/sage/modules/free_module.py

```diff
@@ -3110,6 +3110,122 @@ def hom(self, im_gens, codomain=None, **kwds):
             codomain = R**n
         return super().hom(im_gens, codomain, **kwds)
 
+    def pseudoHom(self, twist, codomain=None):
+        r"""
+        Return the Pseudo Hom space corresponding to given data.
+
+        INPUT:
+
+        - ``twist`` -- the twisting morphism or the twisting derivation
+
+        - ``codomain`` -- (default: ``None``) the codomain of the pseudo
+          morphisms; if ``None``, the codomain is the same than the domain
```

Comment:
```
the same with the domain
```
?

Reply:
I don't understand your comment. The domain is `self` here.

Reply:
I meant the grammar. Isn't it "the same with the domain"?

Reply:
and also "Return the pseudo-Hom space ..."?